packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -133,7 +131,7 @@ export class ProfilesUtils {
             return new imperative.ProfileInfo("zowe", {
                 credMgrOverride: {
                     Manager: credentialManager,
-                    service: credentialManagerMap.credMgrDisplayName,
+                    service: credentialManagerMap.credMgrDisplayName.toString(),
```

Comment:
Adding `.toString` makes the code compile but not sure it has the desired effect. If `credMgrDisplayName` is false, then should we return undefined from this function?

Reply:
addressed as suggested (offline) 😋